README.md

```diff
@@ -45,20 +45,29 @@ TiDB Operator manages [TiDB](https://github.com/pingcap/tidb) clusters on [Kuber
 
     Automatically deploy Prometheus, Grafana for TiDB cluster monitoring.
 
-## Roadmap
+## Quick Start
 
-Read the [Roadmap](./ROADMAP.md).
+You can follow our [Get Started](https://docs.pingcap.com/tidb-in-kubernetes/stable/get-started) guide to quickly start a testing Kubernetes cluster and play with TiDB Operator on your own machine.
 
-## Quick start
+## Documentation
 
-Read the [Quick Start Guide](https://pingcap.com/docs/v3.0/tidb-in-kubernetes/tidb-operator-overview/), which includes all the guides for managing TiDB clusters in Kubernetes.
+You can see our documentation at PingCAP website for more in-depth installation and instructions for production:
 
-## Documentation
+- [English](https://docs.pingcap.com/tidb-in-kubernetes/stable/tidb-operator-overview)
+- [简体中文](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/tidb-operator-overview)
+
+All the TiDB Operator documentation is maintained in the [docs-tidb-operator repository](https://github.com/pingcap/docs-tidb-operator). 
+
+## Community
+
+Feel free to reach out if you have any questions. The maintainers of this project are reachable via:
+
+- [TiDB Community Slack](https://pingcap.com/tidbslack/) in the [#sig-k8s](https://tidbcommunity.slack.com/archives/CHD0HA3LZ) channel
+- [Filling an issue](https://github.com/pingcap/tidb-operator/issue) against this repo
```

Comment:
Is `File an issue` better?

Reply:
"filing an issue" should be better
(if they are concatenated, `are reachable via filing an issue`)

Reply:
OK, then please fix the typo `Filing`.

Reply:
oh, thanks! fixed